src/main/scala/fi/oph/koski/perustiedot/OpiskeluoikeudenPerustiedot.scala

```diff
@@ -70,7 +70,7 @@ object OpiskeluoikeudenPerustiedot {
   def makePerustiedot(row: OpiskeluoikeusRow, henkilöRow: HenkilöRow, masterHenkilöRow: Option[HenkilöRow]): OpiskeluoikeudenPerustiedot = {
     makePerustiedot(
       row.id,
-      row.toOpiskeluoikeusUnsafe,
+      row.toOpiskeluoikeusUnsafe(KoskiSpecificSession.systemUser),
```

Comment:
Tämä ei tunnu täysin turvalliselta: miksi perustiedot pitäisi luoda systeemikäyttäjällä?

Vaikka perustiedoissa ei tällä hetkellä ole mitään sensitiivistä, niin tämä varmasti jää erittäin helposti huomaamatta, jos niihin joskus jokin sellainen kenttä lisätään. En nyt heti keksi selkeämpää ratkaisua, mutta ehkä jokin sellainen voisi olla.

Reply:
Jeh, hyvä huomio, vaihdoin untrustedUseriksi.